documentation/man/features/kw-build.rst

```diff
@@ -102,6 +103,10 @@ OPTIONS
     | **sv** or **vs** enables both.
     | **n** (or any other option) disables notifications (this is the default).
 
+\--from-sha:
```

Comment:
How about expanding the build tutorial by adding this option?
https://kworkflow.org/tutorials/buildlinux.html